tools/logicalimager/tsk_logical_imager.cpp

```diff
@@ -42,6 +42,12 @@ static TSK_TCHAR *progname;
 
 static std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
 
+static void pressAnyKeyToExit(int code) {
+    std::cout << std::endl << "Press any key to exit";
+    (void)getchar();
```

Comment:
It seems like I have to press enter, not any character

Reply:
Fixed